### PR TITLE
Awesome Books ES6 Project

### PR DESCRIPTION
**Pull request by ppmarq1**
Hi!  I kindly ask you for a review of this project. I've completed this milestone where I was required to:

-  Create a new directory and initialize it with git.

- Use "npm init -y" command to create package.json file.
- Create the entry point for JavaScript code called index.js.
- Create index.css file for  styles.
- Create the main index.html file. Link,  index.js, and index.css files.
- Create a directory called modules with 2 linked files (Books.js and Luxon.js).
- Use import modules in the index.js
- Refactor the methods and functions to arrow functions.
- Use Luxon Library.
- Use ES6 syntax.

Kind Regards,

Pedro

### CODE REVIEW

File: modules/Books.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+// Form local storage availability checker function
+export function isStorageAvailable(type) {
+  let storage;
```
Comment (ShoiraTa) [inline modules/Books.js]: - [x] You have done a great job using es6 syntax, the only thing you need to change is using arrow syntax for `isStorageAvailable` function 👍🏻

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @ppmarq1 ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

- Great commit history! ✔️ 
- You passed all the Linters! ✔️ 
- Professional pull request! ✔️ 

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/Books.js (hunk 1 above)
Comment (ppmarq1) [reply modules/Books.js]: Resolved. Thank you for your recommendation!

**Review by t-yanick — APPROVED** [review]
Hi @ppmarq1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**